examples/MQ/pixelAlternative/src/PixelAltFindHits.cxx

```diff
@@ -183,14 +183,12 @@ void PixelAltFindHits::GetParList(TList* tempList)
 {
     fDigiPar = new PixelDigiPar("PixelDigiParameters");
     tempList->Add(fDigiPar);
-
-    return;
 }
 
 void PixelAltFindHits::InitMQ(TList* tempList)
 {
     LOG(info) << "********************************************** PixelAltFindHits::InitMQ()";
-    fDigiPar = (PixelDigiPar*)tempList->FindObject("PixelDigiParameters");
```

Comment:
I think in the Init there should be a dynamic_cast with a check afterwards. We get an object by name which actually can be anything.
 
This statement holds for all usage of "getting by name".

Reply:
Thanks for noting this! You're right!

I added checks for all the ones I have touched.